Formula/kubeless.rb

```diff
@@ -0,0 +1,32 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless/archive/v0.2.3.tar.gz"
+  sha256 "e1b1c3b9498ae42b9f2ca7530506fcb37a136bdffb2af7957c3041a93502972a"
+  bottle :unneeded
+
+  depends_on "go" => :build
+
+  def install
+    commit = "bfefb40"
```

Comment:
What happens if we don't pass the `commit`?

Reply:
when running `kubeless version` it gives you the information about the version (release tag) and the commit of that build. If not given it sets the commit to `HEAD` and that output will differ from the output that the `kubeless` binary we distribute gives.

Reply:
I will move that to a property and cover it with the test